go.sum

```diff
@@ -31,6 +31,7 @@ github.com/Microsoft/hcsshim v0.8.7/go.mod h1:OHd7sQqRFrYd3RmSgbgji+ctCwkbq2wbEY
 github.com/OneOfOne/xxhash v1.2.2/go.mod h1:HSdplMjZKSmBqAxg5vPj2TmRDmfkzw+cTzAElWljhcU=
```

Comment:
Why the changes in `go.sum`?

Try not to commit those changes when developing queries unless really necessary.